packages/calcite-components/src/components/date-picker/date-picker.tsx

```diff
@@ -625,7 +623,13 @@ export class DatePicker extends LitElement {
 
     const startCalendarActiveDate = this.range ? this.activeStartDate : activeDate;
 
-    return <>{this.renderMonth(startCalendarActiveDate, this.maxAsDate, minDate, date, endDate)}</>;
+    return (
+      <>
+        <div ariaHidden={true} class="container" tabIndex="-1">
```

Comment:
nitpick: `tabIndex={-1}` since we're setting the property which expects a number vs the `tabindex` attribute which expects a string.